java/ql/src/utils/model-generator/RegenerateModels.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/python3
+
+# Tool to regenerate existing framework CSV models.
+
+from pathlib import Path
+import json
+import os
+import requests
+import shutil
+import subprocess
+import tempfile
+import sys
+
+
+lgtmSlugToModelFile = {
```

Comment:
Should we consider introducing an invariant instead of an explicit mapping?
The root path could be `java/ql/lib/semmle/code/java/frameworks/` and the slug can then be used to decide subfolder an file name.
eg. slug = "apache/commons-beanutils" -> apache/CommonsBeanutilsGenerated.qll.

This would solve the minor code cohesion issue that we otherwise need to remember to update both the workflow (slug, ref) pair and add an entry into lgtmSlugToModelFile.

Reply:
I used to do that but quickly realized that we have different locations for our models. And yes, we could try and move/rename them all into the same spot but hold back on that. I'm happy either way.

Reply:
Are you thinking about models that has been auto generated and then don't follow this pattern on location or hand written models?

Reply:
Alright. Thank you for the context. If this is undecided, then just leave it as it is :-)